translations/strings.yaml

```diff
@@ -14,6 +14,7 @@ Copied!: ''
 Copy: ''
 Create a customizable syllabus and share it with your students.: ''
 Create a syllabus: ''
+create an account: ''
```

Comment:
It seems that we are starting always in the translations with a capital letter, should we change it to `Create an account`?

Reply:
@Tansito - thanks for the review 👋 

I think we can leave this one as a lowercase, since this particular string starts in the middle of a sentence. Here's the design: 

<img width="1029" alt="Screen Shot 2022-05-02 at 5 23 59 AM" src="https://user-images.githubusercontent.com/6276074/166233151-4725e80e-c8a5-4114-8f1a-e9920c2b5de0.png">